rosdep/python.yaml

```diff
@@ -4950,6 +4950,13 @@ python-typing:
     '*': [python-typing]
     trusty: null
     xenial: null
+python-typing-pip:
+  debian:
+    pip:
+      packages: [typing]
+  ubuntu:
+    pip:
+      packages: [typing]
```

Comment:
@cottsay how does this affect ubuntu/debian distros without python2?
Does `rosdep` already handle this correctly or should the rule not apply to e.g. `focal`?

Reply:
It looks like it will inherit the python version used by the currently sourced ros distro: https://github.com/ros-infrastructure/rosdep/blob/2667916b87c77e9386b93bae93bc843d4c20eab4/src/rosdep2/platforms/pip.py#L49-L68

So we should treat the pip rules as python version independent. On focal, for example, this should always end up as a python 3 rule.